featuretools/entityset/relationship.py

```diff
@@ -65,3 +73,35 @@ def parent_variable(self):
     def child_variable(self):
         """Instance of variable in child entity"""
         return self.child_entity[self._child_variable_id]
+
+    def parent_name(self):
```

Comment:
for discussion: perhaps this should be a `@property`, same with `child_name` to be consistent with `child_variable` and `parent_variable`

Reply:
That makes sense to me. I'm not that familiar with `@property`, but it seems to fit here.

Reply:
ya, let's do it